plugins/scanners/scanoss/src/test/kotlin/ScanOssResultParserTest.kt

```diff
@@ -39,9 +41,44 @@ import org.ossreviewtoolkit.model.SnippetFinding
 import org.ossreviewtoolkit.model.TextLocation
 import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.utils.spdx.SpdxConstants
 import org.ossreviewtoolkit.utils.spdx.SpdxExpression
+import org.ossreviewtoolkit.utils.spdx.SpdxLicenseIdExpression
 
 class ScanOssResultParserTest : WordSpec({
+    "getUniqueLicenseDetails()" should {
```

Comment:
I guess this whole test could be dropped when just using `Collection<SpdxExpression>.toExpression()` instead of introducing this new function.